### PR TITLE
feat: collect ckb unlock signatures without tx verify from verifiers

### PR DESCRIPTION
**Pull request by pygman**

### CODE REVIEW

File: offchain-modules/packages/app-multisign-server/src/ckbSigner.ts [hunk 1]
```diff
@@ -354,6 +367,12 @@ export async function signCkbTx(params: collectSignaturesParams): Promise<SigRes
         return new SigResponse(err);
       }
       break;
+    case 'unlock':
```
Comment (mkxbl) [inline offchain-modules/packages/app-multisign-server/src/ckbSigner.ts:370]: `unlock tx message to sign` is different from `mint tx message to sign`, can not use the same code https://github.com/nervosnetwork/force-bridge/blob/09bb874970dfabe4f36a46cad4e5b0ad4ccfe02d/offchain-modules/packages/app-multisign-server/src/ckbSigner.ts#L361

File: offchain-modules/packages/x/src/multisig/multisig-mgr.ts [hunk 2]
```diff
@@ -52,7 +62,17 @@ export interface ckbCollectSignaturesPayload {
   txSkeleton: TransactionSkeletonObject;
 }
 
-export type collectSignaturesParamsPayload = ethCollectSignaturesPayload | ckbCollectSignaturesPayload;
+export interface ckbUnlockCollectSignaturesPayload {
+  sigType: SigType;
+  unlockRecords?: unlockRecord[];
+  createAssets?: createAsset[];
```
Comment (mkxbl) [inline offchain-modules/packages/x/src/multisig/multisig-mgr.ts:68]: what is `createAssets` field for?

File: offchain-modules/packages/app-multisign-server/src/ckbSigner.ts (hunk 1 above)
Comment (pygman) [reply offchain-modules/packages/app-multisign-server/src/ckbSigner.ts:376]: fixed: switch if sigType is mint or unlock use different code to get the message that to sign

File: offchain-modules/packages/x/src/multisig/multisig-mgr.ts (hunk 2 above)
Comment (pygman) [reply offchain-modules/packages/x/src/multisig/multisig-mgr.ts]: refactored ckbCellCollectSignaturesPayload to mint, createCell, unlock different types, the createAssets field is used only when createCell, moved to ckbCreateCellCollectSignaturesPayload

**Pull request merged**